app/_hub/kong-inc/mtls-auth/1.3-x.md

```diff
@@ -116,23 +130,49 @@ HTTP/1.1 201 Created
+{% navtab Declarative (YAML) %}
+
+To create a subject name mapping using declarative configuration, you will need
+to generate a UUID for each `mtls_auth_credentials` mapping. You can use any
+UUID generator to do this. Here are some common ones, depending on your OS:
+* [Linux](https://man7.org/linux/man-pages/man1/uuidgen.1.html)
+* [MacOS](https://brunty.me/post/uuidgen-on-macos/)
+* [Windows](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/new-guid?view=powershell-7)
+
+Once you have a UUID, add the following to your declarative configuration
+file:
+
+```yaml
+consumers:
+- custom_id: my-consumer
+  username: {consumer}
+  mtls_auth_credentials:
+  - id: bda09448-3b10-4da7-a83b-2a8ba6021f0c
+    subject_name: test@example.com
+```
+
+{% endnavtab %}
+{% endnavtabs %}
 
-form parameter                          | default | description
----                                     | ---     | ---
-`subject_name`<br>*required*            |         | The Subject Alternative Name (SAN) or Common Name (CN) that should be mapped to `consumer` (in that order of lookup).
-`ca_certificate`<br>*optional*          |         | UUID of the Certificate Authority (CA) that the certificate has to be verifiable by for the mapping to success. This is to help distinguish multiple certificates with the same subject name but are issued under different CAs. If empty, the subject name will match certificates issued by any CA under the corresponding `config.ca_certificates`.
+#### Parameters for manual mapping
+
+Form Parameter                            | Default | Description
+---                                       | ---     | ---
+`id`<br>*required for declarative config* |  none   | UUID of the Consumer-mapping. Required if adding mapping using declarative configuration, otherwise generated automatically by Kong's Admin API.
+`subject_name`<br>*required*              |  none   | The Subject Alternative Name (SAN) or Common Name (CN) that should be mapped to `consumer` (in order of lookup).
+`ca_certificate`<br>*optional*            |  none   | **If using the Kong Admin API:** UUID of the Certificate Authority (CA). <br><br> **If using declarative configuration:** Full PEM-encoded CA certificate. <br><br>The provided CA UUID or full certificate has to be verifiable by the issuing certificate authority for the mapping to succeed. This is to help distinguish multiple certificates with the same subject name but are issued under different CAs. <br><br>If empty, the subject name will match certificates issued by any CA under the corresponding `config.ca_certificates`.
 
 ### Matching behaviors
 
-Once a client certificate has been verified as valid, the **Consumer** object will be determined in the following order unless `skip_consumer_lookup` is set to `true`:
+Once a client certificate has been verified as valid, the **Consumer** object will be determined in the following order, unless `skip_consumer_lookup` is set to `true`:
```

Comment:
This could happen more than one time. Avoid future/passive tense.

```suggestion
After a client certificate has been verified as valid, the **Consumer** object is determined in the following order, unless `skip_consumer_lookup` is set to `true`:
```